crfs.go

```diff
@@ -966,11 +969,19 @@ func (h *nodeHandle) ReadDirAll(ctx context.Context) (ents []fuse.Dirent, err er
 //
 // See https://godoc.org/bazil.org/fuse/fs#NodeStringLookuper
 func (n *node) Lookup(ctx context.Context, name string) (fspkg.Node, error) {
+    	if c, ok := n.children[name] ; ok {
+    		return c, nil
+    	}
+
 	e, ok := n.te.LookupChild(name)
 	if !ok {
 		return nil, fuse.ENOENT
 	}
-	return &node{n.fs, e, n.sr, nil}, nil
+	
+    	c := &node{n.fs, e, n.sr, nil, make(map[string]fspkg.Node)}
```

Comment:
Arguably positional initializing was already overdue before, but now it really is. Switch these to named fields here and drop the nil entry.

Reply:
Thank you for your review. I fixed it on b076f18.